sys/fuchsia/restricted

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <zircon/syscalls.h>
+
+# architecture-dependant
```

Comment:
We now have per-arch files:
```
syzkaller$ grep "meta arches" sys/linux/*.txt
sys/linux/dev_kvm.txt:meta arches["386", "amd64", "arm64", "mips64le", "ppc64le", "s390x"]
sys/linux/dev_tlk_device.txt:meta arches["386", "amd64", "arm", "arm64"]
sys/linux/fs_incfs.txt:meta arches["386", "amd64", "arm", "arm64"]
```

If necessary, this can be split per arch.